### PR TITLE
Updated docs for enabling topology

### PR DESCRIPTION
**Pull request by verult**
Documentation for https://github.com/kubernetes-csi/external-provisioner/pull/148. Holding until that PR merges

/hold

### CODE REVIEW

File: book/src/Setup.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ If you want to use the `CSIDriver` CRD and get a preview of how configuration wi
 1) Ensure the feature gate is enabled with `--feature-gates=CSIDriverRegistry=true`
 2) Install the `CSIDriver` CRD on the Kubernetes cluster with the following command:
 ```
-$> kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/testdata/csidriver.yaml
+$> kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/testdata/csidriver.yaml --validate=false
```
Comment (vladimirvivien) [inline book/src/Setup.md]: Thanks for adding this. Missed it.

File: book/src/Setup.md [hunk 2]
```diff
@@ -224,6 +224,20 @@ by default for alpha):
...
 ```
 
+## Topology (alpha)
+In order to support topology-aware dynamic provisioning mechanisms available in Kubernetes, the *external-provisioner* must have the Topology feature enabled:
+
+```
+--feature-gates=Topology=true
+```
+
+In addition, in the *Kubernetes cluster* the `CSINodeInfo` alpha feature must be enabled:
```
Comment (vladimirvivien) [inline book/src/Setup.md]: Add something like `(see above for detail about CSINodeInfo)`

File: book/src/Setup.md [hunk 3]
```diff
@@ -224,6 +224,20 @@ by default for alpha):
 --feature-gates=VolumeSnapshotDataSource=true
 ```
 
+## Topology (alpha)
+In order to support topology-aware dynamic provisioning mechanisms available in Kubernetes, the *external-provisioner* must have the Topology feature enabled:
+
+```
+--feature-gates=Topology=true
```
Comment (leakingtapan) [inline book/src/Setup.md]: Im testing topology support using EBS CSI driver, doesn't seem this feature gate is required for me. And this flag is not present in [kube_features.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go) too. Is this a real feature gate?

**Comment by leakingtapan** [comment]
I happen to submit a similar PR: #65 @lpabon do you want to consolidate the two?

**Comment by leakingtapan** [comment]
I happen to submit a similar PR: #65 @verult do you want to consolidate the two?

File: book/src/Setup.md (hunk 3 above)
Comment (verult) [reply book/src/Setup.md]: This is a provisioner flag

**Comment by verult** [comment]
@leakingtapan yep sounds good, I just have a couple of comments and I'll cherry-pick your PR after they are addressed

**Comment by verult** [comment]
Updated with @leakingtapan 's changes and addressed comments, PTAL

**Comment by leakingtapan** [comment]
/lgtm 
you might need to rebase since my change got merged

**Comment by verult** [comment]
Rebased!

File: book/src/Setup.md (hunk 1 above)
Comment (saad-ali) [reply book/src/Setup.md]: We'll should archive all of this after 1.13 as a 1.12 steps and update it for 1.13.

Comment (verult) [reply book/src/Setup.md]: Yep after 1.13 components roll out we could do another doc pass to make sure everything is updated

**Pull request merged**